js/main.js

```diff
@@ -3,31 +3,72 @@ var summonBtn = document.querySelector('.summon-btn');
 
 var summonImg = document.querySelector('.image');
 
-var acceptBtn = document.querySelector('.accept-btn');
+var acceptBtnDiv = document.querySelector('.accept-btn-div');
+
+var rejectBtn = document.querySelector('.reject-btn');
 
+var acceptBtn = document.querySelector('.accept-btn');
 
-function getLoadFox(){
+var image = '';
 
-  var image = ''
+function getLoadFox() {
 
   var xhr = new XMLHttpRequest();
-  xhr.open('GET', 'https://randomfox.ca/floof/');  // http://shibe.online/api/shibes?count=[1-100]&urls=[true/false]&httpsUrls=[true/false]
+  xhr.open('GET', 'https://randomfox.ca/floof/');
 
   xhr.responseType = 'json';
   xhr.addEventListener('load', function () {
 
-    console.log(xhr.status);
-    console.log(xhr.response);
+    // console.log(xhr.status);
+    // console.log(xhr.response);
 
-    summonImg.setAttribute("src", xhr.response.image)
+    summonImg.setAttribute('src', xhr.response.image);
+    image = xhr.response.image;
 
+    summonBtn.classList.add('hidden');
 
+    acceptBtnDiv.classList.remove('hidden');
 
   });
 
   xhr.send();
 
+}
+
+function rejectFox() {
+  summonBtn.classList.remove('hidden');
+
+  acceptBtnDiv.classList.add('hidden');
+
+  summonImg.setAttribute('src', 'images/pleasesummon.png');
 
 }
 
-summonBtn.addEventListener('click', getLoadFox)
+function acceptFox() {
+  summonBtn.classList.remove('hidden');
+
+  acceptBtnDiv.classList.add('hidden');
+
+  summonImg.setAttribute('src', 'images/pleasesummon.png');
+
+  data.collection.push({ foxImage: image, quote: '' });
+
+}
+
+summonBtn.addEventListener('click', getLoadFox);
+
+rejectBtn.addEventListener('click', rejectFox);
+
+acceptBtn.addEventListener('click', acceptFox);
+
+window.addEventListener('DOMContentLoaded', function (event) {
+  var collectionView = document.querySelectorAll('.collection-img');
+  for (var i = 0; i < data.collection.length; i++) {
+    if (i > 7) {
+      break;
+    } else {
+      collectionView[i].setAttribute('src', data.collection[i].foxImage);
+    }
+
+  }
```

Comment:
This seems like it will always show 6 foxes, even if the user hasn't saved 6 yet. Is that the case? You should only show as many as the user has saved. Rather than having the `img` elements written directly into the HTML, you would likely be better off making a function to generate the DOM tree for all the fox cards. Make sure you make the function `return` the DOM tree so you can append it where the function is called.

Reply:
I was going to code that up in the next feature since the user can only add foxes so far, but not release them.
I can do it for this one though